src/main/kotlin/com/anaplan/engineering/vdmgradleplugin/VdmGradlePlugin.kt

```diff
@@ -173,14 +166,3 @@ internal fun deleteDirectory(directory: File) {
     }
     Files.walkFileTree(directory.toPath(), fileVisitor)
 }
-
-internal fun Project.loadBinarySpecification(binary: File, vararg otherFiles: File): Interpreter {
-    val dialect = project.vdmConfig.dialect
-    val controller = dialect.createController()
-    val parseStatus = controller.parse(listOf(binary) + otherFiles)
-    if (parseStatus != ExitStatus.EXIT_OK) {
-        throw GradleException("VDM parse of generated lib failed")
-    }
-    return controller.getInterpreter()
-}
```

Comment:
Turns out that this was the cause of those `ClassNotFoundException`s about `GradleException`. Which was very confusing, since the error was triggered by a totally unrelated function in this file.